x-pack/plugins/integration_assistant/server/templates/readme.njk

```diff
@@ -0,0 +1,31 @@
+# {{ package_name }} Integration
+
+## Overview
+
+Explain what the integration is, define the third-party product that is providing data, establish its relationship to the larger ecosystem of Elastic products, and help the reader understand how it can be used to solve a tangible problem.
```

Comment:
Not sure if we should keep all this – the idea is that when creating an integration the developer would edit this readme, not that we will show it to the user. Perhaps we should comment out this content?

Reply:
One thing we can do is merge as-is and ask product for suggestions.

Reply:
The content of the readme will change in the [follow up PR](https://github.com/elastic/kibana/pull/194308). The idea is to fine-tuned it with eng, product and support

Reply:
Sounds good